py/desitarget/cuts.py

```diff
@@ -6,6 +6,7 @@
 import sys
 
 import numpy as np
+np.seterr(all="ignore")
```

Comment:
Do we really want to turn all of them off here?  We seem to have come full circle on numpy error masking.  I originally caught and hid the warnings when I knew they were harmless (negative flux is legal but shouldn't pass cuts, so don't jabber about log of a negative number...).  That then got removed to expose all the warnings, and then re-added on a case-by-case basis.  But I think we want to keep the warnings trapping to isolated pieces of code and not a generic module level filter.  Thoughts?